pkgs/applications/misc/lswt/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromSourcehut, nixos, wayland }:
+
+stdenv.mkDerivation rec {
+  pname = "lswt";
+  version = "1.0.4";
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Orwa7sV56AeznEcq/Xj5qj4PALMxq0CI+ZnXuY4JYE0=";
+  };
+
+  makeFlags = [
+    "DESTDIR=${placeholder "out"}"
+    "PREFIX="
+  ];
+
+  meta = with lib; {
+    description = "A command that lists Wayland toplevels";
+    homepage = "https://git.sr.ht/~leon_plickat/lswt";
```

there is a meta/landing page:
```suggestion
    homepage = "https://sr.ht/~leon_plickat/lswt";
```